mk/artifact.mk

```diff
@@ -0,0 +1,57 @@
+USE_PREBUILT ?= 1
+
+CC ?= gcc
+CROSS_COMPILE ?= riscv-none-elf-
+
+BINDIR := $(abspath $(OUT))
+
+TEST_SUITES += \
+	ansibench \
+	rv8-bench
+
+# "ieee754" needs F extension
+# "smolnes", "ticks" have inline assembly and only work in riscv
+TESTBENCHES += \
+	captcha \
+	donut \
+	fcalc \
+	hamilton \
+	$(ieee754) \
+	jit \
+	lena \
+	line \
+	maj2random \
+	mandelbrot \
+	nqueens \
+	nyancat \
+	pi \
+	puzzle \
+	qrcode \
+	richards \
+	rvsim \
+	$(smolnes) \
+	spirograph \
+	$(ticks) \
+	uaes
+
+ifeq ($(USE_PREBUILT),1)
+  LATEST_RELEASE := $(shell wget -q https://api.github.com/repos/sysprog21/rv32emu-prebuilt/releases/latest -O- | grep -Po '(?<="tag_name": ").+(?=",)')
+endif
+
+.PHONY: artifact
+
+artifact:
+ifeq ($(USE_PREBUILT),1)
+	@echo "Fetching prebuilt executables in \"rv32emu-prebuilt\"..."
+	@wget -q --show-progress https://github.com/sysprog21/rv32emu-prebuilt/releases/download/$(LATEST_RELEASE)/rv32emu-prebuilt.tar.gz -O- | tar -C build -xz
+else
+	@$(foreach tb,$(TEST_SUITES), \
+	    git submodule update --init ./tests/$(tb) &&) true
+	@$(foreach tb,$(TEST_SUITES), \
+		$(MAKE) -C ./tests/$(tb) all BINDIR=$(BINDIR) &&) true
+	@$(foreach tb,$(TESTBENCHES), \
+	    $(CC) -m32 -O2 -Wno-unused-result -o $(BINDIR)/linux-x64/$(tb) tests/$(tb).c -lm &&) true
```

Comment:
Is `-m32` required? The binary is placed in `build/linux-x64` directory.

Reply:
If we want to get the more precise result of the performance evaluation between the native execution and the simulation, `-m32` option is required because x86-64 ABI cannot disable the SIMD instruction of the FPU

Reply:
> If we want to get the more precise result of the performance evaluation between the native execution and the simulation, `-m32` option is required because x86-64 ABI cannot disable the SIMD instruction of the FPU

Then, you should clarify the directory name as `linux-x86`. Once a file is built with `-m32`, it would be recognized as the following:
> ELF 32-bit LSB shared object, Intel 80386, version 1 (SYSV)

Reply:
To make it clearer, I'm going to rename it as `linux-x86-soft-float`.

Reply:
Per GCC's naming convention, "softfp" can be applied. That is, linux-x86-softfp.